src/main/java/seedu/address/storage/JsonAdaptedPerson.java

```diff
@@ -185,4 +184,21 @@ public Person toModelType() throws IllegalValueException {
         return new Person(modelDisplayPicture, modelName, modelDescription, modelPhone, modelEmail,
                 modelGithub, modelUniversity, modelMajor, modelFrom, modelTo, modelCap);
     }
+
+    @Override
+    public boolean equals(Object other) {
```

Comment:
`equals` can stay here for now but if it is not utilised later we need to remember to remove it.